### DISCUSSION TITLE
Progress bar

### DISCUSSION THREAD

**Discussion by khshassa** [daboynb/windows_scripts#11] [Ideas]
Replace the download line with 

powershell -NoProfile -Command iwr "https://github.com/daboynb/windows_scripts/archive/refs/heads/main.zip" -outfile "%path_to_use%"\main.zip

then at least the download progress will be visible.

**Comment by daboynb** [comment]
using 
`powershell -command "$wc = New-Object net.webclient; $wc.DownloadFile('https://github.com/daboynb/windows_scripts/archive/refs/heads/main.zip', '%path_to_use%\main.zip'.Replace('path_to_use', '%path_to_use%'))"
powershell -command "Expand-Archive -Path '%path_to_use%\main.zip' -DestinationPath '%path_to_use%' -Force"
`
makes the download faster, sadly with 
`powershell -NoProfile -Command iwr "https://github.com/daboynb/windows_scripts/archive/refs/heads/main.zip" -outfile "%path_to_use%"\main.zip`
it's so slow
